subsys/bluetooth/audio/tbs_internal.h

```diff
@@ -288,51 +288,101 @@ struct bt_tbs_instance {
 
 	uint16_t start_handle;
 	uint16_t end_handle;
+#if defined(CONFIG_BT_TBS_CLIENT_BEARER_PROVIDER_NAME)
 	uint16_t provider_name_handle;
+#endif /* defined(CONFIG_BT_TBS_CLIENT_BEARER_PROVIDER_NAME) */
```

Comment:
We could consider grouping the handles with the `sub_params` to make this less messy. 

@MariuszSkamra also pointed out that if we have `sub_params` for a characteristic, we can even save these 16-bits by only using the `value_handle` of `sub_params`. 

Something to consider for later at least :)